doc/tools/make_rst.py

```diff
@@ -842,6 +872,15 @@ def make_rst_class(class_def: ClassDef, state: State, dry_run: bool, output_dir:
             + "\n\n"
         )
 
+    if class_def.name in CLASSES_WITH_CSHARP_DIFFERENCES:
+        f.write(".. note::\n\n\t")
+        f.write(
+            translate(
+                "There are notable differences when using this API with C#. See :ref:`doc_c_sharp_differences` for more information."
```

Comment:
This should be added to `BASE_STRINGS` here and in https://github.com/godotengine/godot-editor-l10n/blob/main/scripts/extract_classes.py#L29 for `translate` to actually work.

I notice we're also missing the "There is current no description for this ..." strings in `BASE_STRINGS`, if you want to add them too while at it.

Reply:
- I added both and also opened https://github.com/godotengine/godot-editor-l10n/pull/7.